src/TorchSharp/NN/MultiheadAttention.cs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation and contributors.  All Rights Reserved.  See License.txt in the project root for license information.
```

Comment:
This should now say 'Copyright (c) .NET Foundation and Contributors. All Rights Reserved. See LICENSE in the project root for license information.'